### PR TITLE
Make rate limit blocking configurable

### PR DESCRIPTION
**Pull request by rstorey**

### CODE REVIEW

File: concordia/settings_prod.py [hunk 1]
```diff
@@ -78,3 +78,5 @@
 REGISTRATION_SALT = "django_registration"  # doesn't need to be secret
 
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
+
+RATELIMIT_BLOCK = os.getenv("RATELIMIT_BLOCK", True)
```
Comment (acdha) [inline concordia/settings_prod.py]: Doesn’t this need to evaluate that to a Boolean? Maybe just have something like `getenv(…) != “false”`?

Comment (rstorey) [reply concordia/settings_prod.py]: I was thinking we'd set it to 0 or 1 in the env vars

Comment (acdha) [reply concordia/settings_prod.py]: Okay, but don’t we still need to convert it from a string so it won’t always be true?

Comment (rstorey) [reply concordia/settings_prod.py]: Like this? `   RATELIMIT_BLOCK = bool(strtobool(str(os.getenv("RATELIMIT_BLOCK"))))` or you have a different preferred method?

**Comment by rstorey** [comment]
> This entire block can be:
> 
> ```
> RATELIMIT_BLOCK = os.getenv("RATELIMIT_BLOCK", “”).lower() not in (“false”, “0”)
> ```

Thank you - hard to think straight when adrenaline is going.

**Pull request closed without merging**